iio/iio_app/iio_app.c

```diff
@@ -61,6 +61,10 @@
 #include "maxim_irq.h"
 #include "maxim_uart.h"
 #endif
+#if defined(MBED_PLATFORM)
+#include "mbed_irq.h"
+#include "mbed_uart.h"
+#endif
```

Comment:
is this necessary? I don't see any other changes related to these includes in this file

Reply:
I will remove these includes.